src/main/java/org/jitsi/jicofo/JitsiMeetConferenceImpl.java

```diff
@@ -2294,11 +2294,9 @@ boolean handleMuteRequest(Jid fromJid,
             return false;
         }
 
-        // we do not want to remotely mute jigasi as currently
-        // there is no way for it to unmute
-        if (doMute && participant.isSipGateway())
+        if (doMute && (globalConfig.isSipMuteEnabled() == false))
```

Comment:
Tests are failing. There is an error in the logic I think ... Isn't this the correct if statement:
```
if (doMute && participant.isSipGateway() && !globalConfig.isSipMuteEnabled())
```
?

Reply:
You are not checking this whether it is jigasi and you always forbid remote muting, which tests caught.

Reply:
Corrected condition for jigasi